com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -286,7 +286,7 @@ public static void NetworkShow(List<NetworkObject> networkObjects, ulong clientI
                     throw new ArgumentNullException("All " + nameof(NetworkObject) + "s must belong to the same " + nameof(NetworkManager));
                 }
             }
-
+            var networkObjectIdToName = new Dictionary<ulong, string>();
```

Comment:
It would be simpler to just track each individual network object, I'm not sure that doing a batched network object spawn would work as we'd have to allocate somewhere to keep them all in the same place.